moveit_core/CMakeLists.txt

```diff
@@ -21,8 +21,19 @@ endif()
 
 find_package(Boost REQUIRED system filesystem date_time thread iostreams)
 find_package(Eigen3 REQUIRED)
- # TODO: Move collision detection into separate packages
-find_package(Bullet REQUIRED)
+
+# TODO: Move collision detection into separate packages
```

Comment:
What will this entail?

Reply:
This was added as suggested [here](https://github.com/ros-planning/moveit/pull/1504/files#r302605522) by @rhaschke.

> Later, we should factor out FCL and Bullet collision checking into their own packages. Maybe add a todo as a reminder here.

So I guess create stand-alone packages?